var/spack/repos/builtin/packages/ea-utils/package.py

```diff
@@ -13,20 +15,39 @@ class EaUtils(MakefilePackage):
 
     homepage = "https://expressionanalysis.github.io/ea-utils/"
     url = "https://github.com/ExpressionAnalysis/ea-utils/archive/1.04.807.tar.gz"
+    git = "https://github.com/ExpressionAnalysis/ea-utils.git"
+    maintainers = ['snehring']
 
+    version('2021-10-20', commit='10c21926a4dce4289d5052acfd73b8e744d4fede')
```

Comment:
Can we deprecate the older version, so we'll remove it with v0.19 and maintain a unique versioning scheme?

Reply:
Probably. I reached out real quick to see if they'd cut a new release. If you're okay with it I'll let this sit for a few days to see if they respond. Otherwise, I'll move forward with this version scheme.

Reply:
No response from upstream. Deprecated the old version.